lib/task/tools/renameSlugTask.class.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class renameSlugTask extends sfBaseTask
+{
+    protected function configure()
+    {
+        $this->addArguments([
+            new sfCommandArgument('oldSlug', sfCommandArgument::OPTIONAL, 'The slug to update'),
+            new sfCommandArgument('newSlug', sfCommandArgument::OPTIONAL, 'Update name for the slug'),
```

Comment:
Should "slug text" be used rather than "name"? The text "Update name for the slug" could be changed to "Updated slug text".

Reply:
Yep, that does sound better (and I'm realizing that a typo had crept in, It was supposed to say updated).